packages/dapp-svelte-wallet/ui/src/Transaction.svelte

```diff
@@ -25,7 +25,6 @@
     if (typeof outcome !== 'string') {
       outcome = 'Offer was accepted.';
     }
-    alert(outcome);
```

Comment:
Unfortunately I think we need something more complicated to keep encouragement working. Plausible is to only pop up if there was a string return, and then make the faucet not provide a string result value. But hacky. I'll just cope now.

We also should really remember that outcome with the offer in the wallet, so

Reply:
Currently the outcome is being sent back to the encouragement dapp. There's no reason to alert it in the wallet.

Reply:
See https://github.com/Agoric/dapp-encouragement/pull/73 for use of this functionality

Reply:
> See [Agoric/dapp-encouragement#73](https://github.com/Agoric/dapp-encouragement/pull/73) for use of this functionality

... which I've patched not to rely on this PR being merged.